ccc_filter.scpt

```diff
@@ -1,20 +1,44 @@
+set q to "{query}"
+on convertListToString(theList, theDelimiter)
+	set AppleScript's text item delimiters to theDelimiter
+	set theString to theList as string
+	set AppleScript's text item delimiters to ""
+	return theString
+end convertListToString
+
+on convertStringToList(theString, theDelimeter)
```

Comment:
Can we use AppleScript's [`paragraph`](https://developer.apple.com/library/content/documentation/LanguagesUtilities/Conceptual/MacAutomationScriptingGuide/ManipulateText.html#//apple_ref/doc/uid/TP40016239-CH33-SW20) instead of this?